packages/fixie/src/main.ts

```diff
@@ -48,6 +48,38 @@ function registerDeployCommand(command: Command) {
     });
 }
 
+/** Run an agent locally. */
+function registerServeCommand(command: Command) {
+  command
+    .command('serve [path]')
+    .description('Run an agent locally')
```

Comment:
Once the agent is running, where do we want to direct people to interact with it? (e.g. "am I supposed to CURL this thing?") I think we want to tell people to go to beta.fixie.ai?

Reply:
This happens already:

```
🥡 Revision 5467ec8e was deployed to https://beta.fixie.ai/agents/mdw/sidekick-github
```